pkg/dogstatsd/server.go

```diff
@@ -327,7 +325,6 @@ func NewServer(aggregator *aggregator.BufferedAggregator, extraTags []string) (*
 		eolTerminationUDP:         eolTerminationUDP,
 		eolTerminationUDS:         eolTerminationUDS,
 		eolTerminationNamedPipe:   eolTerminationNamedPipe,
-		telemetryEnabled:          telemetry_utils.IsEnabled(),
```

Comment:
Removing this unused global seems non-problematic, but unrelated.  Was this mixed in by mistake?

Reply:
It seems it was part of my stash. This is deadcode anyway but I can create a PR if you prefer

Reply:
Meh, might as well leave it in :)